apps/demos/src/app/features/tutorials/basics/2-input-bindings/input-bindings.start.component.ts

```diff
@@ -101,7 +102,7 @@ export class InputBindingsStart extends RxState<ComponentState> implements OnIni
   @Input()
   set refreshInterval(refreshInterval: number) {
     if (refreshInterval > 4000) {
-      this._refreshInterval = refreshInterval;
+      this._refreshInterval = refreshInterval; // Partially update the state with the state slice
```

Comment:
If there is a "helper comment" for the person doing the tutorial mostly it's a good idea to put the comment above.
Also a nice thing we can do here is use emoticons to highlight a specific piece of code e.g. 👇
```suggestion
      this._refreshInterval = refreshInterval; // Partially update the state with the state slice
```